### DISCUSSION TITLE
Are there any ways to install on linux

### DISCUSSION THREAD

**Discussion by byteshiva** [lima-vm/lima#1291] [Q&A] [answered]
Are there any ways to install LIMA on linux

**Comment by afbjorklund** [comment, marked as answer]
Sure, it will install the same way as on Mac ?

https://github.com/lima-vm/lima#installation

1) Homebrew package

2) Manual installation

It is also possible to build from source, using `make`

----

You want to have KVM enabled, for acceleration

Hopefully, some distributions will also include it:

https://repology.org/project/lima/versions

Maybe they are waiting for "1.0", or something ?

**Comment by afbjorklund** [comment]
You could also install `nerdctl` and friends directly on your own computer, instead of in a VM.
